packages/manager/modules/pci/src/projects/new/config/routing.js

```diff
@@ -71,6 +74,11 @@ export default /* @ngInject */ ($stateProvider) => {
           type: 'action',
         });
       },
+
+      returnToPreviousPage: ($window) => () => {
+        console.log('ZM:: history', $window.history);
```

Comment:
```suggestion
```

Reply:
> Hello @mohammed-zahaf, any reason for targeting master branch? Can it be taken with the next sprint release?
> 
> Jeremy

Yep, because all tickets that i am working on are part of a special project that we have to deploy them as soon as possible.